### ISSUE TITLE
Feast UI doesn't work with SQLRegistry

### ISSUE THREAD

**Issue by 0fair** [feast-dev/feast#2963]
## Expected Behavior 
Feast UI works correctly
I can see all Data Sources / Feature Views etc

## Current Behavior
Nothing is show
Errors in console:
```
react_devtools_backend.js:4026 ZodError: [
  {
    "code": "invalid_type",
    "expected": "string",
    "received": "undefined",
    "path": [
      "featureServices",
      0,
      "meta",
      "createdTimestamp"
    ],
    "message": "Required"
  }
]
    at yk (index.mjs:449:23)
    at n.value (index.mjs:561:16)
    at n.value (index.mjs:541:29)
    at useLoadRegistry.ts:35:47
```

## Steps to reproduce
build feature_store.yaml with SQLRegistry
deploy features to registry via "feast apply"
start webserver with "feast ui"
go to browser and try feast ui

### Specifications

- Version: 0.22.1
- Platform: arm64
- Subsystem: macOS

## Possible Solution

insert create_timestamp columns into sql registry schema and write code to add it to registry proto
now there is no columns like "created_timestamp" in sql registry

**Comment by adchia** [comment]
cc @achals

**Comment by achals** [comment]
Should be fixed by #3033 too
